Sources/App/Commands/Analyzer.swift

```diff
@@ -3,26 +3,26 @@ import Vapor
 import ShellOut
 
 
-struct InspectorCommand: Command {
+struct AnalyzerCommand: Command {
```

Comment:
I went with Anaylzer for now to sort of indicate that it's a "heavier" operation. Let me know if you've got better ideas!

Reply:
There's no ideal names, but I think having one start with an `A` and one with an `I` will at least make it easier to keep a mental model of which does what 😂